src/components/marker/timeline-preview.tsx

```diff
@@ -0,0 +1,230 @@
+      </Fragment>
+    );
+  }
+
+  private _getData(): any {
+    const initialData: any = {[ItemTypes.QuizQuestion]: [], [ItemTypes.Hotspot]: [], [ItemTypes.AnswerOnAir]: []};
+    const data = this.props.cuePointsData.reduce((acc, cp) => {
+      return {
+        ...acc,
+        [cp.type]: [...acc[cp.type], cp]
+      }
+    }, initialData);
+
+    return {
+      hotspots: data[ItemTypes.Hotspot],
+      quizQuestions: data[ItemTypes.QuizQuestion],
+      answerOnAir: data[ItemTypes.AnswerOnAir]
+    };
+  }
+
+  private _renderArrowButton() {
+    const isNavigationPluginOpen = this.props.isNavigationPluginOpen();
+    const isSmallPlayer = this.props.playerSize! === PLAYER_SIZE.SMALL;
+    return (
+      <Fragment>
+        <button className={[styles.markerLink, !isNavigationPluginOpen ? styles.disabled : '', isSmallPlayer ? styles.smallPlayer : ''].join(' ')}
+                onClick={this._handleClick} data-testid="previewArrowButton">
+          <Icon size={IconSize.medium} name={'arrowClose'}/>
+        </button>
+        <button className={[styles.markerLink, isNavigationPluginOpen ? styles.disabled : '', isSmallPlayer ? styles.smallPlayer : ''].join(' ')}
+                onClick={this._handleClick} data-testid="previewArrowButton">
+          <Icon size={IconSize.medium} name={'arrowOpen'}/>
+        </button>
+      </Fragment>
+    );
+  };
+
+  private _renderSmallPlayerHeader(relevantChapter: Chapter | undefined, data: any) {
+    const {quizQuestions, hotspots, answerOnAir} = data;
+    const isSmallPlayer = this.props.playerSize! === PLAYER_SIZE.SMALL;
+    const titleClassName = [styles.titleWrapper, isSmallPlayer ? styles.smallPlayer : ''].join(' ');
+    const itemsWrapperClassName = [styles.itemsWrapper, isSmallPlayer ? styles.smallPlayer : ''].join(' ');
+
+    return (
+      <div className={[styles.header, styles.smallPlayer].join(' ')}>
+        <div className={itemsWrapperClassName} data-testid="cuePointPreviewHeaderItems">
+        {relevantChapter && <Title iconName={'chapter'} shouldDisplayTitle={false} className={titleClassName} />}
+        {hotspots.length > 0 && <Title iconName={'hotspot'} shouldDisplayTitle={false} className={titleClassName} />}
+        {quizQuestions.length > 0 && <Title iconName={'quiz'} shouldDisplayTitle={false} className={titleClassName} />}
+        {answerOnAir.length > 0 && <Title iconName={'answerOnAir'} shouldDisplayTitle={false} className={titleClassName} />}
+        </div>
+        {this.props.shouldRenderArrowButton() && this._renderArrowButton()}
+      </div>
+    );
+  }
+
+  onMouseMove = (e: MouseEvent) => {
+    // prevent the preview from moving with the mouse
+    e.stopPropagation();
+  };
+
+  private _shouldRenderHeader(relevantChapter: Chapter | undefined): boolean {
+    return this.props.playerSize !== 'small' && (this.props.cuePointsData.length > 0 || !relevantChapter?.isDummy!);
```

Comment:
I believe player has enums for playerSize
check KalturaPlayer.ui.Components.PLAYER_SIZE

Reply:
actually you already have it in component `PLAYER_SIZE.SMALL`

Reply:
oops, missed that one. thanks!